specification/cognitiveservices/data-plane/AutoSuggest/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  package-name: "@azure/cognitiveservices-autosuggest"
+  output-folder: "$(typescript-sdks-folder)/packages/@azure/cognitiveservices-autosuggest"
+  override-client-name: AutoSuggestAPIClient
```

Comment:
Could you remove this? We prefer the name to be AutoSuggestClient

Reply:
Sure